### DISCUSSION TITLE
[Bug]: Synchronization of external website meetings

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#665] [General]
### Contact Details

ecomm.cer@aamail.org

### Website With Issue

https://alcoholics-anonymous.eu/

### What happened?

Geneva has recently implemented the meeting list. 
They provided me with the key. 
When I synchronised, the meetings were all put under Geneva as a country, instead of Switzerland, Geneva. 
So we worked with their webmaster and he created the top region as Switzerland. 
I re-synched, but this didnt help. 
I also deleted all the meetings from that region, started from scratch. 
Nothing is working. 
Please advise. 

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome

**Comment by tech2serve** [comment]
I just tested using Safari and Chrome on my iPad, and I got this:

![58BE79AF-47F5-4BBC-A829-24C22800E35C](https://user-images.githubusercontent.com/31904316/151841755-d06d5384-5346-4a31-84b3-53f0c8fd6962.jpeg)

Is this not what you’re wanting/expecting? 

Did you clear your browser cache?

**Comment by julien-paris** [comment]
Hi, 
Yes cache is cleared. 
That is what i left on the site for the users. those meeting were manually entered. 
Please see the video (from the staging site)
In this case, they are enetered but are are neither added to Switzerland > Geneva, nor is the new region created. 
This is another case we had the other day, but not the only one. 
https://www.loom.com/share/f09cdf7fee5b44f6a8c3d38510aa0f15

**Reply by tech2serve** [reply]
Being able to see the feed would help. 

If you prefer to keep the details more private, send them to devs@code4recovery.org. 

When I get them, I can try to duplicate the issue.

**Reply by joshreisner** [reply]
hi julien, i'm a little confused. in this loom you import 27 meetings from geneva, and then go to monday and see 5. is that not what you expected?

**Reply by julien-paris** [reply]
Sent by mail
